Upgrade to Drupal 10.6, prepare for Drupal 11

vinmassaro opened the pull request:
### Description of work
- Updates `drupal/core` to Drupal 10.6.8
- Updates many contrib modules. Some have been removed from this list if they don't impact the interface:
  - [ ] [address](https://drupal.org/project/address) 2.0.2 -> 2.0.4
  - [ ] [admin_toolbar](https://drupal.org/project/admin_toolbar) 3.5.0 -> 3.6.3
  - [ ] [anchor_link](https://drupal.org/project/anchor_link) 3.0.0-beta1 -> 3.0.4
  - [ ] [auto_entitylabel](https://drupal.org/project/auto_entitylabel) 3.3.0 -> 3.4
  - [ ] [better_exposed_filters](https://drupal.org/project/better_exposed_filters) 6.0.5 -> 7.1.1
  - [ ] [calendar_link](https://drupal.org/project/calendar_link) 3.0.3 -> 3.0.4
  - [ ] [cas](https://drupal.org/project/cas) 2.3.2 -> 3.0.1
  - [ ] [components](https://drupal.org/project/components) 3.1.0 -> 3.2.0
  - [ ] [config_filter](https://drupal.org/project/config_filter) 2.6.0 -> 2.7.0
  - [ ] [config_ignore](https://drupal.org/project/config_ignore) 3.3.0 -> 3.4.0
  - [ ] [editor_advanced_link](https://drupal.org/project/editor_advanced_link) 2.2.6 -> 2.3.4
  - [ ] [editoria11y](https://drupal.org/project/editoria11y) 2.1.19 -> 3.0.3
  - [ ] [entity_usage](https://drupal.org/project/entity_usage) 2.0-beta24 -> 2.0.0-beta29
  - [ ] [environment_indicator](https://drupal.org/project/environment_indicator) 4.0.22 -> 4.0.25
  - [ ] [gin](https://drupal.org/project/gin) 3.0.0-rc13 -> 4.1.3
  - [ ] [gin_lb](https://drupal.org/project/gin_lb) 1.0.0-rc8 -> 2.0.0-beta1
  - [ ] [gin_moderation_sidebar](https://drupal.org/project/gin_moderation_sidebar) 1.0.0 -> 1.0.1
  - [ ] [google_tag](https://drupal.org/project/google_tag) 2.0.8 -> 2.0.9
  - [ ] [honeypot](https://drupal.org/project/honeypot) 2.1.4 -> 2.2.2
  - [ ] [improve_line_breaks_filter](https://drupal.org/project/improve_line_breaks_filter) 1.6 -> 1.7
  - [ ] [jquery_ui](https://drupal.org/project/jquery_ui) 1.7.0 -> 1.8.0
  - [ ] [jquery_ui_datepicker](https://drupal.org/project/jquery_ui_datepicker) 2.1.0 -> 2.1.1
  - [ ] [layout_builder_browser](https://drupal.org/project/layout_builder_browser) 1.7 -> 1.9.0
  - [ ] [layout_builder_restrictions](https://drupal.org/project/layout_builder_restrictions) 3.0.0 -> 3.0.4
  - [ ] [libraries](https://drupal.org/project/libraries) 4.0.5 -> 4.1.1
  - [ ] [linkit](https://drupal.org/project/linkit) 6.1.4 -> 7.0.14
  - [ ] [mailchimp_transactional](https://drupal.org/project/mailchimp_transactional) 1.1.2 -> 1.2.0
  - [ ] [markup](https://drupal.org/project/markup) 2.0.0 -> 2.0.1
  - [ ] [maxlength](https://drupal.org/project/maxlength) 3.1.0 -> 3.1.3
  - [ ] [media_entity_download](https://drupal.org/project/media_entity_download) 2.3.0 -> 2.4.0
  - [ ] [media_file_delete](https://drupal.org/project/media_file_delete) 1.3.0 -> 1.3.1
  - [ ] [media_library_edit](https://drupal.org/project/media_library_edit) 3.0.4 -> 3.0.5
  - [ ] [media_library_form_element](https://drupal.org/project/media_library_form_element) 2.1.1 -> 2.1.4
  - [ ] [media_thumbnails_pdf](https://drupal.org/project/media_thumbnails_pdf) 2.0 -> 2.0.1
  - [ ] [menu_admin_per_menu](https://drupal.org/project/menu_admin_per_menu) 1.5 -> 1.7
  - [ ] [menu_breadcrumb](https://drupal.org/project/menu_breadcrumb) 2.0.0-alpha0 -> 2.0.1
  - [ ] [menu_item_limit](https://drupal.org/project/menu_item_limit) 2.1.0 -> 2.2.0
  - [ ] [metatag](https://drupal.org/project/metatag) 2.0.2 -> 2.2.0
  - [ ] [migrate_plus](https://drupal.org/project/migrate_plus) 6.0.4 -> 6.0.8
  - [ ] [migrate_tools](https://drupal.org/project/migrate_tools) 6.0.4 -> 6.1.4
  - [ ] [multiple_fields_remove_button](https://drupal.org/project/multiple_fields_remove_button) 2.2 -> 2.3
  - [ ] [multivalue_form_element](https://drupal.org/project/multivalue_form_element) 1.0-beta6 -> 1.0.0-beta8
  - [ ] [node_revision_delete](https://drupal.org/project/node_revision_delete) 2.0.0-rc1 -> 2.1.1
  - [ ] [override_node_options](https://drupal.org/project/override_node_options) 2.8.0 -> 2.9.0
  - [ ] [paragraphs](https://drupal.org/project/paragraphs) 1.18.0 -> 1.20.0
  - [ ] [paragraphs_features](https://drupal.org/project/paragraphs_features) 2.0.0 -> 2.2.0
  - [ ] [pathauto](https://drupal.org/project/pathauto) 1.13.0 -> 1.15.0
  - [ ] [recaptcha_v3](https://drupal.org/project/recaptcha_v3) 2.0.3 -> 2.0.5
  - [ ] [redirect](https://drupal.org/project/redirect) 1.10.0 -> 1.13
  - [ ] [role_delegation](https://drupal.org/project/role_delegation) 1.2 -> 1.6.0
  - [ ] [search_api](https://drupal.org/project/search_api) 1.35.0 -> 1.40.0
  - [ ] [search_api_exclude](https://drupal.org/project/search_api_exclude) 2.0.2 -> 2.0.3
  - [ ] [simple_sitemap](https://drupal.org/project/simple_sitemap) 4.2.2 -> 4.2.3
  - [ ] [smart_date](https://drupal.org/project/smart_date) 4.1.6 -> 4.2.5
  - [ ] [taxonomy_manager](https://drupal.org/project/taxonomy_manager) 2.0.19 -> 2.0.23
  - [ ] [twig_tweak](https://drupal.org/project/twig_tweak) 3.4.0 -> 3.4.2
  - [ ] [typogrify](https://drupal.org/project/typogrify) 1.4-rc1 -> 1.5.0
  - [ ] [webform](https://drupal.org/project/webform) 6.2.7 -> 6.3.0-beta9
  - [ ] [workflow_buttons](https://drupal.org/project/workflow_buttons) 1.0.0-beta6 -> 1.0-beta7

 - Also fixes yalesites-org/YaleSites-Internal#1244
 
### Post-release todos:

single_content_sync:
1. Upgrade single_content_sync after this PR is released
1. Reinstate https://github.com/yalesites-org/yalesites-project/commit/03d1d8f01e9b40bfc1bf980cd86725b03fdd37c6
1. Re-export starterkit for compatibility once module update is released

config_filter
1. Verify config_filter no longer used and remove from composer.json

vinmassaro committed:
feat: upgrade drupal/core to 10.4.6

vinmassaro committed:
feat: upgrade drush to 13.x

vinmassaro committed:
chore: disable deprecated layout_builder_expose_all_field_blocks and actions_ui modules

vinmassaro committed:
feat: upgrade to drupal/fast_404:3.4

vinmassaro committed:
feat: upgrade to drupal/pantheon_advanced_page_cache:2.3.2

vinmassaro committed:
feat: upgrade to yalesites-org/yale_cas:v1.1.0

vinmassaro committed:
fix: update cas module config to be compliant with cas 3.x

vinmassaro committed:
feat: upgrade to drupal/cas:3.0.0-beta4

vinmassaro committed:
feat: upgrade drupal/core to 10.4.7

vinmassaro committed:
feat: upgrade to drupal/mailchimp_transactional:1.1.4

vinmassaro committed:
feat: upgrade to drupal/pantheon_secrets:1.0.6

vinmassaro committed:
feat: upgrade to drupal/single_content_sync:1.4.13

vinmassaro committed:
feat: upgrade to drupal/webform:6.3.0-beta2

vinmassaro committed:
feat: upgrade to drupal/role_delegation:1.3

vinmassaro committed:
feat: upgrade to drupal/search_api_exclude:2.0.3

vinmassaro committed:
feat: upgrade to drupal/workflow_buttons:1.0-beta7

vinmassaro committed:
Revert "feat: upgrade to drupal/single_content_sync:1.4.13"

This reverts commit 1e14cf18e32995bafb6b4e3ff9aad922c850d6ae.

vinmassaro committed:
feat: upgrade to drupal/single_content_sync:1.4.13
Also updates from Simplefield to GenericField due to change in https://www.drupal.org/project/single_content_sync/issues/3498506.

vinmassaro committed:
fix: enable js_cookie dependency for CAS module
See https://www.drupal.org/project/cas/issues/3528901 for more info on why we need this now, and will remove it later.

vinmassaro committed:
Revert "feat: upgrade to drupal/single_content_sync:1.4.13"

This reverts commit c4f37944e19ad40a56145318ada95ad02a0e6c49.

vinmassaro committed:
feat: upgrade drupal/core to 10.5.1

vinmassaro committed:
feat: upgrade to drupal/better_exposed_filters:6.0.6

vinmassaro committed:
feat: upgrade to drupal/admin_toolbar:3.6.1

vinmassaro committed:
chore: config syntax update

vinmassaro committed:
feat: upgrade to drupal/address:2.0.4

vinmassaro committed:
feat: upgrade to drupal/simple_sitemap:4.2.2

vinmassaro committed:
feat: upgrade to drupal/typogrify:1.4

vinmassaro committed:
feat: upgrade to drupal/anchor_link:3.0.3

vinmassaro committed:
feat: upgrade to drupal/auto_entitylabel:3.4

vinmassaro committed:
feat: upgrade to drupal/calendar_link:3.0.4

vinmassaro committed:
feat: upgrade to drupal/captcha:2.0.7

vinmassaro committed:
feat: upgrade to drupal/honeypot:2.2.2

vinmassaro committed:
feat: upgrade to drupal/improve_line_breaks_filter:1.7

vinmassaro committed:
feat: upgrade to drupal/layout_builder_browser:1.8

vinmassaro committed:
feat: upgrade to drupal/menu_admin_per_menu:1.7

vinmassaro committed:
feat: upgrade to drupal/gin_moderation_sidebar:1.0.1

vinmassaro committed:
feat: upgrade to drupal/multiple_fields_remove_button:2.3

vinmassaro committed:
feat: upgrade to drupal/multivalue_form_element:1.0.0-beta7

vinmassaro committed:
feat: upgrade to drupal/menu_breadcrumb:2.0.0

vinmassaro committed:
feat: upgrade to drupal/node_revision_delete:2.0.0

vinmassaro committed:
feat: upgrade to drupal/override_node_options:2.9.0

vinmassaro committed:
feat: upgrade to drupal/paragraphs_features:2.2.0

vinmassaro committed:
feat: upgrade to drupal/editor_advanced_link:2.3.1

vinmassaro committed:
Merge branch 'develop' into drupal-10.5

vinmassaro committed:
feat: upgrade drupal/core to 10.5.3

vinmassaro committed:
chore: remove drupal/cas patch that is no longer needed

vinmassaro committed:
chore: remove laminas/laminas-escaper since we no longer use drupal/bugherd

vinmassaro committed:
chore: remove ckeditor-5-anchor-drupal from composer.json
This library is now installed directly by drupal/anchor_link.

vinmassaro committed:
fix: upgrade to build-tools-ci to 9.x with PHP 8.3

vinmassaro committed:
fix: remove ckeditor-5-anchor-drupal repository from root composer.json

vinmassaro committed:
chore: remove northernco/ckeditor5-anchor-drupal

vinmassaro committed:
feat: upgrade to drupal/linkit:7.0.9

vinmassaro committed:
Merge remote-tracking branch 'origin/develop' into drupal-10.5

vinmassaro committed:
feat: upgrade drupal/core to 10.5.4

vinmassaro committed:
feat: upgrade to gin:4.1.1, gin_lb:2.0.0-beta1

vinmassaro committed:
feat: upgrade to better_exposed_filters:7.1.0-beta3

vinmassaro committed:
chore: disable google_analytics to prep for future removal

vinmassaro committed:
chore: re-export config

vinmassaro committed:
style: alphabetize modules patches in profile composer.json

vinmassaro committed:
feat: upgrade drupal/core to 10.5.5

vinmassaro committed:
feat: upgrade to drupal/menu_item_limit:2.2.0 and remove D11 compatibility patch

vinmassaro committed:
feat: upgrade to yalesites-org/ai_engine:1.2.13

vinmassaro committed:
chore: uninstall upgrade_status to re-add for local config_split

vinmassaro committed:
feat: make ys_admin_theme D11 compatible

vinmassaro committed:
Merge branch 'develop' into drupal-10.5

vinmassaro committed:
chore: re-export config with minor changes

vinmassaro committed:
feat: upgrade to drupal/captcha:2.0.9

vinmassaro committed:
feat: upgrade to drupal/better_exposed_filters:7.1.0-beta4

vinmassaro committed:
feat: upgrade to drupal/admin_toolbar:3.6.2

vinmassaro committed:
feat: upgrade to drupal/editoria11y:2.2.18

vinmassaro committed:
feat: upgrade to drupal/devel_kint_extras:1.1.6

vinmassaro committed:
feat: upgrade to drupal/editor_advanced_link:2.3.2

vinmassaro committed:
feat: upgrade to drupal/devel:5.4.0

vinmassaro committed:
feat: upgrade to drupal/config_filter:2.7.0

vinmassaro committed:
feat: upgrade to drupal/environment_indicator:4.0.25

vinmassaro committed:
feat: upgrade to drupal/fast_404:3.6.0

vinmassaro committed:
feat: upgrade to drupal/entity_usage:2.0.0-beta25

vinmassaro committed:
feat: upgrade to drupal/gin:4.1.2

vinmassaro committed:
feat: upgrade to drupal/google_analytics:4.0.3

vinmassaro committed:
feat: upgrade to drupal/google_tag:2.0.9

vinmassaro committed:
feat: upgrade to drupal/layout_builder_restrictions:3.0.4

vinmassaro committed:
feat: upgrade to drupal/jquery_ui:1.8.0

vinmassaro committed:
feat: upgrade to drupal/jquery_ui_datepicker:2.1.1

vinmassaro committed:
feat: upgrade to drupal/libraries:4.1.1

vinmassaro committed:
feat: upgrade to drupal/markup:2.0.1

vinmassaro committed:
feat: upgrade to drupal/linkit:7.0.11

vinmassaro committed:
feat: upgrade to drupal/mailchimp_transactional:1.2.0

vinmassaro committed:
feat: upgrade to drupal/media_library_form_element:2.1.4

vinmassaro committed:
feat: upgrade to drupal/maxlength:3.1.3

vinmassaro committed:
feat: upgrade to drupal/menu_breadcrumb:2.0.1

vinmassaro committed:
feat: upgrade to drupal/media_thumbnails_pdf:2.0.1

vinmassaro committed:
feat: upgrade to drupal/media_entity_download:2.4.0

vinmassaro committed:
feat: upgrade to drupal/migrate_plus:6.0.8

vinmassaro committed:
feat: upgrade to drupal/node_revision_delete:2.0.1

vinmassaro committed:
feat: upgrade to drupal/metatag:2.2.0

vinmassaro committed:
feat: upgrade to drupal/migrate_tools:6.1.2

vinmassaro committed:
feat: upgrade to drupal/pantheon_advanced_page_cache:2.3.4

vinmassaro committed:
feat: upgrade to drupal/paragraphs:1.19.0

vinmassaro committed:
feat: upgrade to drupal/recaptcha_v3:2.0.4

vinmassaro committed:
feat: upgrade to drupal/redirect:1.12.0

vinmassaro committed:
feat: upgrade to drupal/pathauto:1.14.0

vinmassaro committed:
feat: upgrade to drupal/role_delegation:1.4.0

vinmassaro committed:
feat: upgrade to drupal/redis:1.11.0

vinmassaro committed:
feat: upgrade to drupal/search_api:1.40.0

vinmassaro committed:
feat: upgrade to drupal/single_content_sync:1.4.14

vinmassaro committed:
feat: upgrade to drupal/upgrade_status:4.3.8

vinmassaro committed:
feat: upgrade to drupal/webform:6.3.0-beta5

vinmassaro committed:
feat: upgrade to drupal/taxonomy_manager:2.0.22

vinmassaro committed:
feat: upgrade to drupal/twig_tweak:3.4.1

vinmassaro committed:
feat: upgrade to drupal/smart_date:4.2.4

vinmassaro committed:
chore: remove merged taxonomy_manager patch (3514624)

vinmassaro committed:
chore: update patch #3057250 for drupal/redirect:1.12.0

vinmassaro committed:
chore: remove google_analytics patches for future module removal

vinmassaro committed:
chore: remove old media_library_form_element patch (#3277273)

vinmassaro committed:
chore: update migrate_plus plugin patch (#3040427)

vinmassaro committed:
feat: upgrade drupal/core to 10.5.6

vinmassaro committed:
chore: remove devel_kint_extras
devel_kint_extras was removed in devel 5.4.0.

vinmassaro committed:
chore: set php_runtime_generation for future PHP 8.4 support

vinmassaro committed:
feat: upgrade to drupal/editor_advanced_link:2.3.3

vinmassaro committed:
Merge branch 'develop' into drupal-10.5

vinmassaro committed:
Merge branch 'develop' into drupal-10.5

vinmassaro committed:
feat: upgrade to drupal/multivalue_form_element:1.0.0-beta8

vinmassaro committed:
chore: revert upgrade to single_content_sync

vinmassaro committed:
feat: upgrade to drupal/field_group:4.0.0

vinmassaro committed:
chore: remove drupal/entity_usage

vinmassaro committed:
Merge branch 'develop' into drupal-10.5

vinmassaro committed:
Merge branch 'develop' into drupal-10.5

vinmassaro committed:
chore: re-export config

vinmassaro committed:
Merge branch 'develop' into drupal-10.5

vinmassaro committed:
chore: register SA-CONTRIB-2025-060 for composer audit

vinmassaro committed:
feat: upgrade to drupal/devel:5.5.0

vinmassaro committed:
feat: upgrade to drupal/components:3.2.0

vinmassaro committed:
feat: upgrade to drupal/better_exposed_filters:7.1.1

vinmassaro committed:
feat: upgrade to drupal/webform:6.3.0-beta6

vinmassaro committed:
feat: upgrade to drupal/single_content_sync:1.4.14

vinmassaro committed:
feat: upgrade to drupal/simple_sitemap:4.2.3

vinmassaro committed:
feat: upgrade drupal/core to 10.5.7

vinmassaro committed:
Merge branch 'develop' into drupal-10.5

vinmassaro committed:
Revert "feat: upgrade to drupal/single_content_sync:1.4.14"

This reverts commit 5faaa36a6a0a0a7be09e584a38a1cc09b79e24fd.

vinmassaro committed:
chore: remove imagemagick fix script

vinmassaro committed:
feat: upgrade drupal/core to 10.5.8

vinmassaro committed:
feat: upgrade to drupal/linkit:7.0.12

vinmassaro committed:
Merge branch 'develop' into drupal-10.5

vinmassaro committed:
Merge branch 'develop' into drupal-10.5

vinmassaro committed:
chore: remove unused allow-plugin definition for drupal/console-extend-plugin

vinmassaro committed:
chore: cleanup

vinmassaro committed:
chore: update config

vinmassaro committed:
feat: upgrade drupal/core to 10.6.0

vinmassaro committed:
feat: upgrade drupal/core to 10.6.1

vinmassaro committed:
feat: upgrade drupal/core to 10.6.2

vinmassaro committed:
Merge branch 'develop' into drupal-10.5

vinmassaro committed:
feat: update drupal/role_delegation to 1.5.0

vinmassaro committed:
Merge branch 'develop' into drupal-10.5

vinmassaro committed:
Merge branch 'develop' into drupal-10.5

vinmassaro committed:
Merge branch 'develop' into drupal-10.5

vinmassaro committed:
Merge branch 'develop' into drupal-10.5

vinmassaro committed:
feat: upgrade drupal/core to 10.6.5

vinmassaro committed:
Merge branch 'develop' into drupal-10.5

vinmassaro committed:
fix: pin drupal/redirect to 1.x-dev so patches apply

vinmassaro committed:
feat: upgrade to drupal/cas:3.0.1

vinmassaro committed:
Revert "fix: enable js_cookie dependency for CAS module"

This reverts commit ef998ed9efbac146fb848ec4f6d5701bae2c66e7.

vinmassaro committed:
chore: remove composer security temp fix now that lando is updated

vinmassaro committed:
feat: upgrade to drupal/admin_toolbar:3.6.3

vinmassaro committed:
feat: upgrade to drupal/anchor_link:3.0.4

vinmassaro committed:
feat: upgrade to drupal/editor_advanced_link:2.3.4

vinmassaro committed:
feat: upgrade to drupal/fast_404:3.7.0

vinmassaro committed:
feat: upgrade to drupal/devel_kint_extras:1.1.6

vinmassaro committed:
feat: upgrade to drupal/editoria11y:2.2.20

vinmassaro committed:
feat: upgrade to drupal/linkit:7.0.13

vinmassaro committed:
feat: upgrade to drupal/media_library_edit:3.0.5

vinmassaro committed:
feat: upgrade to drupal/gin:4.1.3

vinmassaro committed:
feat: upgrade to drupal/role_delegation:1.6.0

vinmassaro committed:
feat: upgrade to drupal/node_revision_delete:2.0.2

vinmassaro committed:
feat: upgrade to drupal/migrate_tools:6.1.3

vinmassaro committed:
feat: upgrade to drupal/paragraphs:1.20.0

vinmassaro committed:
feat: upgrade to drupal/typogrify:1.5.0

vinmassaro committed:
feat: upgrade to drupal/taxonomy_manager:2.0.23

vinmassaro committed:
feat: upgrade to drupal/upgrade_status:4.3.9

vinmassaro committed:
feat: upgrade to drupal/webform:6.3.0-beta8

vinmassaro committed:
chore: update config

vinmassaro committed:
Merge branch 'develop' into drupal-10.5

vinmassaro committed:
feat: upgrade drupal/core to 10.6.7

vinmassaro committed:
feat: upgrade to drupal/editoria11y:2.2.22

vinmassaro committed:
feat: upgrade to drupal/layout_builder_browser:1.9.0

vinmassaro committed:
feat: upgrade to drupal/node_revision_delete:2.0.3

vinmassaro committed:
Merge branch 'develop' into drupal-10.5

vinmassaro committed:
feat: upgrade to drupal/redirect:1.13 and remove patch from #3057250

vinmassaro committed:
feat: upgrade to drupal/linkit:7.0.14

vinmassaro committed:
feat: upgrade to drupal/config_ignore:3.4.0

vinmassaro committed:
feat: upgrade to drupal/node_revision_delete:2.1.1

vinmassaro committed:
feat: upgrade to drupal/recaptcha_v3:2.0.5

vinmassaro committed:
feat: upgrade to drupal/smart_date:4.2.5

vinmassaro committed:
feat: upgrade to drupal/twig_tweak:3.4.2

vinmassaro committed:
chore: update node_revision_delete config

vinmassaro committed:
Merge branch 'develop' into drupal-10.5

vinmassaro committed:
Merge branch 'develop' into drupal-10.5

vinmassaro committed:
feat: export updated config for drupal/editoria11y

vinmassaro committed:
feat: upgrade drupal/core to 10.6.8

vinmassaro committed:
feat: upgrade to drupal/entity_usage:2.0.0-beta29

vinmassaro committed:
feat: upgrade to drupal/pathauto:1.15.0

vinmassaro committed:
feat: upgrade to drupal/migrate_tools:6.1.4

vinmassaro committed:
feat: upgrade to drupal/webform:6.3.0-beta9

dblanken-yale commented on web/profiles/custom/yalesites_profile/config/sync/webform.settings.yml:
```diff
@@ -153,11 +153,11 @@ element:
   default_empty_option_optional: ''
+  default_algolia_places_app_id: ''
+  default_algolia_places_api_key: ''
   excluded_elements:
     password: password
     password_confirm: password_confirm
-  default_algolia_places_app_id: ''
-  default_algolia_places_api_key: ''
```
Is this residual from another db instance like when we did the news troubleshooting?

vinmassaro replied:
This is from config that ships with webform: https://git.drupalcode.org/project/webform/-/blob/6.3.x/config/schema/webform.settings.schema.yml#L387-394

dblanken-yale replied:
Oh wow that's surprising to see!  Is what it is.  :D

dblanken-yale reviewed (commented):
Double checking this one; I remember we did some troubleshooting with news on algolia, so wondered if this was a residual from that instance?

vinmassaro committed:
chore: re-export config for pathauto

vinmassaro committed:
feat: upgrade to drupal/editoria11y:3.0.3

dblanken-yale committed:
fix(1244): add Chosen to affiliation filter

Include 'field_affiliation_target_id' in the list of form elements that
use the Chosen library on views exposed forms. This improves the user
experience for filtering by affiliation.

vinmassaro committed:
feat: upgrade drupal/core to 10.6.9

vinmassaro committed:
feat: upgrade to yalesites-org/yale_cas:v1.3.0

vinmassaro committed:
feat: upgrade to drupal/better_exposed_filters:7.1.2

vinmassaro committed:
feat: upgrade to drupal/linkit:7.0.15

vinmassaro committed:
chore: update linkit patch for drupal/linkit:7.0.15

vinmassaro committed:
Merge remote-tracking branch 'origin/develop' into drupal-10.5

vinmassaro committed:
chore: remove composer audit ignore entries that were needed before ^10.6

vinmassaro committed:
feat: update how jjj/chosen is installed
oomphinc/composer-installers-extender is now abandoned, so we need to reference the jjj/chosen repository in root composer.json.

vinmassaro committed:
chore: re-export config

vinmassaro committed:
Merge branch 'develop' into drupal-10.5
